xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESettings.cpp

```diff
@@ -183,9 +186,66 @@ void CActiveAESettings::SettingOptionsAudioDevicesFillerGeneral(
       if (StringUtils::EqualsNoCase(current, sink->second))
         foundValue = true;
     }
+
+    // partial match of device name OR device frienly name in same driver
+    if (!foundValue)
+    {
+      for (const auto& sink : sinkList)
+      {
+        if (sink.second.find(dev.driver) == std::string::npos)
+          continue;
```

Comment:
not a problem in Windows due to the very different strings, but maybe sink.second should be parsed/split first for a more robust comparison with driver or name.
same note about sink.first used a bit later

Reply:
This is also solved with the last commit